examples/light-switch-app/nrfconnect/main/include/BindingHandler.h

```diff
@@ -42,9 +37,24 @@ class BindingHandler
         bool IsGroup{ false };
     };
 
+    void Init();
+    void PrintBindingTable();
+    bool IsGroupBound();
+
+    static void SwitchWorkerHandler(intptr_t);
+    static void OnInvokeCommandFailure(chip::DeviceProxy * aDevice, const BindingData & aBindingData, CHIP_ERROR aError);
+
+    static BindingHandler & GetInstance()
+    {
+        static BindingHandler sBindingHandler;
+        return sBindingHandler;
+    }
+
 private:
     static void OnOffProcessCommand(chip::CommandId, const EmberBindingTableEntry &, chip::DeviceProxy *, void *);
     static void LevelControlProcessCommand(chip::CommandId, const EmberBindingTableEntry &, chip::DeviceProxy *, void *);
     static void LightSwitchChangedHandler(const EmberBindingTableEntry &, chip::DeviceProxy *, void *);
     static void InitInternal(intptr_t);
+
+    bool mCaseSessionRecovered = false;
```

Comment:
But binding data are related to the specific cluster and command, so it will not work if somebody send different commands one by one. I believe we should store this flag for a specific node ID, but for now we support only one node ID for unicast traffic, so I didn't care much about it.

Reply:
Ok, but now if there are two simultaneous commands sent and the CASE is lost, won't the second one be simply ignored? Anyway, probably it would require more work to make such scenarios work correctly, so nevermind.